lib/puppet/type/dism.rb

```diff
@@ -52,15 +40,4 @@
     #defaultto([0, 3010])
     defaultto([0, 3010, 3010 & 0xFF])
   end
-
-  def munge_boolean(value)
-    case value
-      when true, "true", :true
-        :true
-      when false, "false", :false
-        :false
-      else
-        fail("munge_boolean only takes booleans")
-    end
-  end
```

Comment:
I believe the middle one is the preferred method, and after a cursory test it appears that this PR needs updating. Oops.